### PR TITLE
Update calculator.py

### PR DESCRIPTION
**Pull request by abirhossain11**

### CODE REVIEW

**Comment by sunniraleigh** [comment]
Thanks for correcting my indents and catching the 0 division error!

**Review by abirhossain11 — COMMENTED** [review]
you are most welcome

**Comment by abirhossain11** [comment]
> Thanks for correcting my indents and catching the 0 division error!



> Thanks for correcting my indents and catching the 0 division error!

File: calculator.py [hunk 1]
```diff
@@ -10,39 +10,44 @@
...
 
         # if division
         elif tokens[0] == "/" or tokens[0] == "divide":
-            print(divide(float(tokens[1]), float(tokens[2])))
+          try:
+            ans = divide(float(tokens[1]), float(tokens[2]))
+          except ZeroDivisionError:
+            print("You can't divide by 0!")
+          else:
+            print(ans)
```
[the remark this answers is not in the document]
Comment (abirhossain11) [reply calculator.py:37]: Can you change

**Comment by abirhossain11** [comment]
please merge

**Comment by abirhossain11** [comment]
please merge

**Comment by sunniraleigh** [comment]
I'm not going to merge the pull request because this is my course work. I'll move you recommendations to an issue in issues and return to them on my own time.

**Pull request closed without merging**